src/Runner/TestInstance.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+/**
+ * This file is part of the Nette Tester.
+ * Copyright (c) 2009 David Grudl (https://davidgrudl.com)
+ */
+
+namespace Tester\Runner;
+
+
+/**
+ * Carries information about a single test instance.
+ */
+class TestInstance
```

Comment:
What about constants `Runner::FAILED, ::PASSED, ::SKIPPED`? This is maybe better place for them.